### ISSUE TITLE
--exclude-other-sites broken

### ISSUE THREAD

**Issue by dlg0** [drush-ops/drush#48]
Hi, thanks for the great work on Drush. I'm using 6.0 installed via pear. I'm trying to core-rsync from a local development site to a remote live site, all existing within a multisite install of D7. The help says to use the "--exclude-other-sites" option to sync just one site. However, it seems to be broken with the following error thrown ...

$ drush rsync @mlq-dev @mlq-live --exclude-other-sites
You will destroy data from mylifequotes.us:/var/www/drupal/ and replace with data from /var/www/drupal//
Do you really want to continue? (y/n): y
rsync: --exclude-other-sites-processed: unknown option
rsync error: syntax or usage error (code 1) at main.c(1453) [client=3.0.9]
Could not rsync from /var/www/drupal// to                                    [error]

so basically an "--exclude-other-sites-processed" flag is getting passed through to rsync. If I instead do a 

$ drush rsync @mlq-dev:sites/mylifequotes.us/ @mlq-live:sites/mylifequotes.us/

it works fine. However, this is only a workaround.

Some discussion of this is in a bug registered on drupal.org

https://drupal.org/node/1244384

but it's marked as closed although it seems to persist.

Thanks.
